cypress/integration/browse-projects-page.spec.js

```diff
@@ -32,4 +71,32 @@ describe('browse projects page tests', () => {
         })
```

Comment:
Would like to refactor the test above now that they are using mocked data along the lines below, but am seeing a curious result where the tag takes effect but the list is not actually filtered?  Any suggestions/help tracking down where that filtering takes place much appreciated
```js
  it('filters projects by language', () => {
    const selectedLanguageFilter = 'JavaScript';
    cy.get(`input[value="${selectedLanguageFilter}"]`)
      .check({ force: true })
      .get('.usa-card :contains("Languages")')
      .parent()
      .should('contain.text', selectedLanguageFilter)
  })
```